Applications/FileIO/Legacy/OGSIOVer4.cpp

```diff
@@ -475,32 +475,39 @@ bool readGLIFileV4(const std::string& fname,
     else
         INFO("GeoLib::readGLIFile(): tag #POLYLINE not found.");
 
+    if (!ply_vec->empty())
+        geo.addPolylineVec(
+            std::move(ply_vec), unique_name,
+            std::move(ply_names));  // KR: insert into GEOObjects if not empty
```

Comment:
The comment just says what is already clear from the code. But what does the comment mean?